server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_14__add_datasource_sync_status.sql

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+create table if not exists connect_connection_sync_history(
```

Comment:
4.3.2.14 is occupied, 4.3.2.19 is available

Reply:
done